lib/services/elasticsearch.js

```diff
@@ -1411,32 +1452,34 @@ class ElasticSearch extends Service {
       }
     }
 
-    return this.mGet(index, collection, toDelete)
-      .then(({ hits }) => {
+    // @todo should be done in the clientAdapter,
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)